### DISCUSSION TITLE
python manage.py migrate betiğinde hata alıyorum

### DISCUSSION THREAD

**Discussion by ERNCNR** [realsuayip/django-sozluk#78] [General]
python manage.py migrate komutunu çalıştırdığımda 
`django.core.exceptions.ImproperlyConfigured: settings.DATABASES is improperly configured. Please supply the ENGINE value. Check settings documentation for more details.`
hatasını alıyorum. settings.py'den database türünü değiştirdim denedim yine aynı hatayı veriyor. Sebebi nedir?

**Comment by realsuayip** [comment]
Docker setup'u kullanmıyor olabilir misiniz?

**Comment by ERNCNR** [comment]
Docker'sız, https://github.com/realsuayip/django-sozluk/blob/master/docs/turkish/installation.md burada anlattığınız yöntem ile, debian vps'imde deploy etmeye çalışıyorum.

**Comment by realsuayip** [comment]
Docker kullanmanızı öneririm. environment variable'leriniz düzgün okunamamış dosyalardan anlaşıldığı üzere.

**Comment by yesnt10** [comment]
Aynı hatayı ben de alıyorum. Docker ve readmede yer alan tüm bileşenlerin kurulumunu yaptım.
